src/main/java/mate/academy/rickandmorty/dto/external/CharacterResponseDataDto.java

```diff
@@ -0,0 +1,20 @@
+package mate.academy.rickandmorty.dto.external;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+public record CharacterResponseDataDto(
+        int id,
+        String name,
+        String status,
+        String species,
+        String type,
+        String gender,
+        OriginResponseDto origin,
+        OriginResponseDto location,
+        String image,
+        ArrayList<String> episode,
```

Comment:
No need to use particular List implementation
```suggestion
        List<String> episode,
```